db/fixtures/tactical_vote_stt_recs.rb

```diff
@@ -33,7 +33,7 @@ def load
       lookup[party.smv_code.to_sym] = party
     end
 
-    non_tory_parties = Party.all.select { |p| p.smv_code != "con"}
+    included_parties = Party.where.not({ smv_code: [:con, :reform] }).all
```

Comment:
Just to make it clear that we remain non-partisan, might be worth including a comment here that this is not any kind of subjective decision or editorialising by us, it's simply translating the meaning of `:heart` which STT intends into our data model.